utils/esr_peer_info.cpp

```diff
@@ -401,35 +404,45 @@ namespace
                 + pad_str_to_mod_64(string_to_hex(swarm_id))// hexified swarm id
         };
     }
+}
 
 
+namespace bzn::utils::esr
+{
     // TODO: replace this with a function that uses the ABI to create the request data
+    // data_string_for_get_peer_info has been moved out of the anonymous namespace to make it possible to
+    // unit test directly.
     const std::string
     data_string_for_get_peer_info(const std::string &swarm_id, const std::string &peer_id)
     {
+        static const off_t PARAMETER_OFFSET{64};
```

Comment:
move to unnamed namespace

Reply:
All statics moved to anonymous namespace where appropriate